front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -91,6 +92,7 @@ const StdcmConsist = ({ disabled = false }: StdcmConfigCardProps) => {
   };
 
   const onSelectSuggestion = (option?: LightRollingStockWithLiveries) => {
+    prefillConsist(option);
```

Comment:
You need to pass the select towed rolling stock here (if there is one), otherwise the inputs are not updated correctly